integration/test-integration/src/test/resources/testng-cc-with-apim.xml

```diff
@@ -83,12 +81,6 @@
             <class name="org.wso2.choreo.connect.tests.setup.withapim.CcShutdownExecutor"/>
         </classes>
     </test>
-    <test name="cc-specific-tests-with-apim" parallel="false">
-        <classes>
-            <class name="org.wso2.choreo.connect.tests.setup.withapim.CcStartupExecutorTwo"/>
-            <class name="org.wso2.choreo.connect.tests.testcases.withapim.APIKeyHeaderTestCase"/>
```

Comment:
This was added to create CC-with-APIM deployment considering a changed internal-key name.
I.e. to have the internal-key name similar to the API key name. Are we going to cover that in another new test case  added with this PR ?

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2501/commits/bfd4a9a3442475d917b8d577820f91c50a473793

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2501/commits/bfd4a9a3442475d917b8d577820f91c50a473793